homeassistant/components/dynalite/__init__.py

```diff
@@ -198,6 +198,36 @@ async def async_setup(hass: HomeAssistant, config: Dict[str, Any]) -> bool:
             )
         )
 
+    def get_bridges(host: str) -> List[DynaliteBridge]:
+        result = []
+        for entry_id in hass.data[DOMAIN]:
+            cur_bridge = hass.data[DOMAIN][entry_id]
+            if not host or cur_bridge.host == host:
+                result.append(cur_bridge)
+        return result
+
+    async def request_area_preset_service(service_call: ServiceCall):
+        host = service_call.data.get(CONF_HOST, "")
+        bridges = get_bridges(host)
+        LOGGER.debug("Selected bridged for service call: %s", bridges)
+        area = service_call.data[CONF_AREA]
+        channel = service_call.data.get(CONF_CHANNEL)
+        for bridge in bridges:
+            bridge.dynalite_devices.request_area_preset(area, channel)
+
+    hass.services.async_register(
+        DOMAIN,
+        "request_area_preset",
+        request_area_preset_service,
+        vol.Schema(
+            {
+                vol.Optional(CONF_HOST): cv.string,
+                vol.Required(CONF_AREA): int,
+                vol.Optional(CONF_CHANNEL): int,
```

Comment:
We only use `CONF_*` constants for config items. For service data items we use `ATTR_*`.
```suggestion
                vol.Optional(ATTR_HOST): cv.string,
                vol.Required(ATTR_AREA): int,
                vol.Optional(ATTR_CHANNEL): int,
```